tutoroars/plugin.py

```diff
@@ -241,32 +238,23 @@
 ########################################
 # CUSTOM JOBS (a.k.a. "do-commands")
 ########################################
+# Ex: "tutor dev do load-xapi-test-data"
+@click.command()
+def load_xapi_test_data() -> list[tuple[str, str]]:
```

Comment:
This is totally the best place for this command. Great idea, @bmtcril ! 
FYI I tweaked it a little and added docs for it with 4cce6b9c0a723682a3006357546c9ce3831c0653.

Reply:
Perfect, that was on my todo list for post merge so thanks for grabbing it!